pkg/reconciler/ingress/ingress.go

```diff
@@ -75,85 +69,49 @@ var (
 
 // Reconciler implements the control loop for the Ingress resources.
 type Reconciler struct {
-	*reconciler.Base
+	*spkgreconciler.Base
 
 	istioClientSet       istioclientset.Interface
 	virtualServiceLister istiolisters.VirtualServiceLister
 	gatewayLister        istiolisters.GatewayLister
 	secretLister         corev1listers.SecretLister
 	ingressLister        listers.IngressLister
 
-	configStore pkgreconciler.ConfigStore
-	tracker     tracker.Interface
-	finalizer   string
+	tracker   tracker.Interface
+	finalizer string
 
 	statusManager status.Manager
 }
 
 var (
-	_ controller.Reconciler                = (*Reconciler)(nil)
+	_ ingressreconciler.Interface          = (*Reconciler)(nil)
+	_ ingressreconciler.Finalizer          = (*Reconciler)(nil)
 	_ coreaccessor.SecretAccessor          = (*Reconciler)(nil)
 	_ istioaccessor.VirtualServiceAccessor = (*Reconciler)(nil)
 )
 
+func newReconciledNormal(namespace, name string) pkgreconciler.Event {
+	return pkgreconciler.NewEvent(corev1.EventTypeNormal, "IngressTypeReconciled", "IngressType reconciled: \"%s/%s\"", namespace, name)
+}
```

Comment:
FWIW, I haven't been doing this in Serving.  I'd suggest that we omit for consistency and add it everywhere if it's what we want.